ml-notebook/packages.txt

```diff
@@ -205,15 +189,15 @@ fribidi==1.0.10
 frozenlist==1.4.1
 fsspec==2024.6.0
 future==1.0.0
-gast==0.5.4
+gast==0.5.5
 gcc==12.3.0
 gcc_impl_linux-64==12.3.0
 gcc_linux-64==12.3.0
 gcm_filters==0.3.0
 gcsfs==2024.6.0
 gdal==3.9.0
 gdk-pixbuf==2.42.12
-geocube==0.5.2
+geocube==0.5.3
 geographiclib==2.0
 geopandas==0.14.4
```

Comment:
still not 1.0, so maybe some other package is pinning <1

Reply:
Tracking issue at https://github.com/conda-forge/geocube-feedstock/pull/40